_aggregations/pipeline/avg-bucket.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Average bucket
+parent: Pipeline aggregations
+nav_order: 10
+redirect_from:
+  - /query-dsl/aggregations/pipeline-agg#avg_bucket-sum_bucket-min_bucket-max_bucket/
+  - /query-dsl/aggregations/pipeline/avg-bucket/
+---
+
+# Average bucket aggregations
+
+The `avg_bucket` aggregation is a sibling aggregation that calculates the average of a metric in each bucket of a previous aggregation.
+
+The specified metric must be numeric and the sibling aggregation must be a multi-bucket aggregation.
+
+## Parameters
+
+The `avg_bucket` aggregation takes the following parameters.
+
+| Parameter             | Required/Optional | Data type       | Description |
+| :--                   | :--               |  :--            | :--         |
+| `buckets_path`        | Required          | String          | The path of the aggregation buckets to be aggregated. See [Pipeline aggregations]({{site.url}}{{site.baseurl}}/aggregations/pipeline/index#pipeline-aggregation-syntax). |
+| `gap_policy`          | Optional          | String          | The policy to apply to missing data. Valid values are `skip`, `insert_zeros`, and `keep_values`. Default is `skip`. |
```

Comment:
While `skip` and `insert_zeros` seem intuitive, I am wondering what `keep_values` mean?

Reply:
`keep_values` is not a valid parameter. Removing.